### DISCUSSION TITLE
How to set .env path

### DISCUSSION THREAD

**Discussion by aspraveen** [adonisjs/core#1594] [Help] [answered]
Can anyone advise how to load **.env** file from another directory in Adonis 5 ? 

I tried old documentation

Different .env file
At times you may want to load a different .env file stored on your server at a different location. Same can be done as follows.

ENV_PATH=/user/.env node server.js

but not figuring out where to set it?

**Comment by prashanth1k** [comment]
Did [changing metaFiles](https://preview.adonisjs.com/guides/http/static-files/#using-a-custom-directory) not help?

**Comment by thetutlage** [comment, marked as answer]
`ENV_PATH=/user/.env node server.js` should work

**Reply by aspraveen** [reply]
Thanks Virk!

To be more precise for future reference, 

Modified package.json file

"start": "node ace serve --watch",

to

"start": "ENV_PATH=/pathto/.env node ace serve --watch",

**Comment by sergefabre** [comment]
Hello
I failed to use this command : 

> 'ENV_PATH' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes

an idea ?
